docs/design.md

```diff
@@ -83,58 +81,67 @@ A storage pod restarts can be expensive in terms of requiring data rebuilding an
   * physical location - node, rack, zone
   ```
   apiVersion: openebs.io/v1alpha1
-  kind: Disk
+  kind: BlockDevice
   metadata:
-    name: disk-3d8b1efad969208d6bf5971f2c34dd0c
+    name: blockdevice-b528112c024fec6e8ecb20af2638493d
     labels:
-      "kubernetes.io/hostname": "gke-openebs-user-default-pool-044afcb8-bmc0"
+      kubernetes.io/hostname: gke-akhil-ndm-pool-1-4349c998-2kd9
   status:
-    state: Active
+      claimState: Unclaimed
+      state: Active
   spec:
     capacity:
-      logicalSectorSize: 512
-      storage: 53687091200
+      storage: 402653184000
     details:
+      compliance: SPC-4
+      deviceType: disk
+      driveType: SSD
       firmwareRevision: '1   '
-      model: PersistentDisk
-      serial: disk-2
-      compliance: "SPC-4"
+      hardwareSectorSize: 4096
+      logicalBlockSize: 4096
+      model: EphemeralDisk
+      physicalBlockSize: 4096
+      serial: local-ssd-0
       vendor: Google
     devlinks:
     - kind: by-id
       links:
-      - /dev/disk/by-id/scsi-0Google_PersistentDisk_disk-2
-      - /dev/disk/by-id/google-disk-2
+      - /dev/disk/by-id/scsi-0Google_EphemeralDisk_local-ssd-0
+      - /dev/disk/by-id/google-local-ssd-0
     - kind: by-path
       links:
-      - /dev/disk/by-path/virtio-pci-0000:00:03.0-scsi-0:0:2:0
+      - /dev/disk/by-path/pci-0000:00:04.0-scsi-0:0:1:0
+    filesystem:
+      fsType: ext4
+      mountPoint: /mnt/disks/ssd0
+    nodeAttributes:
+      nodeName: gke-akhil-ndm-pool-1-4349c998-2kd9
     path: /dev/sdb
   ```
 
-- node-disk-manager can also be made to auto-provision disks by using a DiskClaim CR similar to (PVC that uses dynamic provisioners to create a PV). As an example, if the node-disk-manager is running on a Kubernetes Cluster in AWS, it can initiate a request to the underlying AWS to create a EBS and attach to the node. This functionality will be access controlled.
+- node-disk-manager can also be made to auto-provision disks by using a BlockDeviceClaim CR similar to (PVC that uses dynamic provisioners to create a PV). As an example, if the node-disk-manager is running on a Kubernetes Cluster in AWS, it can initiate a request to the underlying AWS to create a EBS and attach to the node. This functionality will be access controlled.
 
-- The User or storage-operators( like OpenEBS provisioner) can use the kube-api/kubectl (like `kubectl get disks`) to get information on all the disks in the Cluster.
+- The User or storage-operators( like OpenEBS provisioner) can use the kube-api/kubectl (like `kubectl get blockdevices`) to get information on all the blockdevices in the Cluster.
 
 - The node-disk-manager, can be used to create StoragePools by taking one or more Disk CRs and putting a storage layer on top of it like ext4, lvm, zfs or spin-up containers like OpenEBS cStor Pool or Ceph OSD (Confirm this), etc., The StoragePool creation can be triggered via an StoragePoolClaim that the user can feed in via kubectl. Examples:
```

Comment:
@kmova  These can be removed from the design right? as storage pools are no longer included in the scope of NDM.

Reply:
yes @akhilerm

Reply:
Done.